netshare/models/doppelganger_tf/dataset.py

```diff
@@ -75,15 +75,16 @@ def data_loader(image_buffer, running_flag, files, transform, config):
             if not image_buffer.empty():
                 time.sleep(0.5)
                 continue
-            file_id = np.random.choice(len(files))
-            image = np.load(files[file_id])
-            image_ = {}
-            for k in image.files:
-                image_[k] = image[k]
-            image_ = transform(image_, config)
             try:
+                file_id = np.random.choice(len(files))
```

Comment:
print informative errors when the process fails.
Before that - it failed quietly and the process got stuck